app/webrtc/client/actionLink.js

```diff
@@ -0,0 +1,29 @@
+import { Session } from 'meteor/session';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import toastr from 'toastr';
+
+import { actionLinks } from '../../action-links/client';
+import { APIClient } from '../../utils/client';
+import { Messages } from '../../models';
+
+actionLinks.register('joinLivechatWebRTCCall', function(message) {
+	if (Session.get('openedRoom')) {
```

Comment:
I didn't understand the use of this statement, could you please explain what its doing and why is it required?

Reply:
I think I can remove it because, in Jitsi, it was used to get the room id, and here I am using the room id from the message.rid instead of Session.get('openedRoom').